interface/patient_tracker/patient_tracker.php

```diff
@@ -158,9 +158,7 @@ function refreshbegin(first){
       refreshme();
     }
     setTimeout("refreshbegin('0')",parsetime);
-  <?php } else { ?>
```

Comment:
I just removed the return because a 'return' with a return value is meaningless php.  So it behaves the same way it did before.